bin/sentieon-genomics-202010.02/lib/python/sentieon/vcflib/vcf.py

```
@@ -0,0 +1,465 @@
+        return VCFReader(self, chrom, start, end)
+
+    def emit(self, v):
+        if v.line is None:
+            self.format(v)
+        self.fp.write(v.line.encode() + b'\n')
+        self.index.add(v.chrom, v.pos, v.end, self.fp.tell())
+
+    def cmp_variants(self, v1, v2):
+        v1_contig_idx = list(self.contigs.keys()).index(v1.chrom)
+        v2_contig_idx = list(self.contigs.keys()).index(v2.chrom)
+        if not cmp(v1_contig_idx, v2_contig_idx):
```

Ruff (0.8.2)

`cmp` may be undefined, or defined from star imports

(F405)